jcl/src/java.base/share/classes/java/lang/ClassLoader.java

```diff
@@ -1206,7 +1206,7 @@ public final Module getUnnamedModule()
 final Class<?> loadClass(Module module, String className) {
 	Class<?> localClass = null;
 	try {
-		localClass = loadClassHelper(className, false, false, module);
+		localClass = loadClassHelper(className, false, true, module);
```

Comment:
The method comment needs updating to match.

Reply:
Updated.